cli/lib/kontena/cli/nodes/ssh_command.rb

```diff
@@ -4,28 +4,47 @@ class SshCommand < Kontena::Command
     include Kontena::Cli::GridOptions
 
     parameter "NODE_ID", "Node id"
+    parameter "[COMMANDS] ...", "Run command on host"
+
     option ["-i", "--identity-file"], "IDENTITY_FILE", "Path to ssh private key"
     option ["-u", "--user"], "USER", "Login as a user", default: "core"
     option "--private-ip", :flag, "Connect to node's private IP address"
     option "--internal-ip", :flag, "Connect to node's internal IP address (requires VPN connection)"
 
+    requires_current_master
+    requires_current_grid
+
     def execute
-      require_api_url
-      require_current_grid
-      token = require_token
+      node = client.get("nodes/#{current_grid}/#{node_id}")
+
+      provider = node["labels"].find{ |l| l.start_with?('provider=')}.to_s.split('=').last
```

Comment:
Looks like recent kontena-server is also returning nodes with `"labels": null"`:

```
NoMethodError: undefined method `find' for nil:NilClass
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/nodes/ssh_command.rb:20:in `execute'
  /home/kontena/kontena/kontena/cli/lib/kontena/command.rb:187:in `run'
  /home/kontena/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
  /home/kontena/kontena/kontena/cli/lib/kontena/command.rb:187:in `run'
  /home/kontena/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
  /home/kontena/kontena/kontena/cli/lib/kontena/command.rb:187:in `run'
  /home/kontena/kontena/kontena/cli/vendor/bundle/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/command.rb:132:in `run'
  bin/kontena:16:in `<top (required)>'
```